packages/mobile/src/store/auth/selectors.ts

```diff
@@ -0,0 +1,3 @@
+import { RootState } from '~/common/types/types';
+
+export const getIsLoggedIn = (state: RootState) => state.auth.isLoggedIn;
```

Comment:
Please name all selectors with `selector` prefix. And let's use something like `const selectCurrentUser = (state: RootState) => state.auth.user` instead to keep reducer simple

Reply:
corrected